code/modules/antagonists/cult/cult_comms.dm

```diff
@@ -7,8 +7,21 @@
 
 	buttontooltipstyle = "cult"
 	check_flags = AB_CHECK_INCAPACITATED|AB_CHECK_HANDS_BLOCKED|AB_CHECK_IMMOBILE|AB_CHECK_CONSCIOUS
+	/// If the cult action does not require hands to be performed
+	var/does_not_need_hands = FALSE
+	/// If the cult action can be performed while immobile
+	var/does_not_need_mobility = FALSE
 	ranged_mousepointer = 'icons/effects/mouse_pointers/cult_target.dmi'
 
+/datum/action/innate/cult/New()
+	..()
+	// If we dont need hands, remove the check from check_flags
+	if(does_not_need_hands == TRUE)
+		check_flags = check_flags & ~AB_CHECK_HANDS_BLOCKED
+	// If we dont need to be mobile, remove the check from check_flags
+	if(does_not_need_mobility == TRUE)
+		check_flags = check_flags & ~AB_CHECK_IMMOBILE
```

Comment:
Why didn't you just remove those check_flags in the definitions of those two actions? 😵

Reply:
I dunno, do you want me to do that instead?

Reply:
That'd be better, yes.

Reply:
done, thanks